src/Box.js

```diff
@@ -4,11 +4,38 @@ import PropTypes from 'prop-types'
 
 const align = responsiveStyle('text-align', 'align')
 
+
```

Comment:
While you are in this file, would you mind also update the code format to remove the extra new line per linting warning? 🙏 
<img width="531" alt="screen shot 2017-10-10 at 7 01 22 pm" src="https://user-images.githubusercontent.com/6441326/31414750-7f33ddf4-aded-11e7-9aa3-122e0d826837.png">